packages/blade/src/utils/testId/testId.native.ts

```diff
@@ -0,0 +1,5 @@
+const testId = (defaultKey: string): { testID: string } => ({
```

Comment:
Capitalise ID?
```suggestion
const testID = (defaultKey: string): { testID: string } => ({
```

Reply:
Should we do something else for this like we did for blade-old [here](https://github.com/razorpay/blade/blob/master/packages/blade-old/src/_helpers/automation-attributes.web.js)

We might want to add more meta data and could reuse such functions

Reply:
Yeah we can even have a common function for this. like how we did mapA11yProps. But lets not do it this PR, we'll pick it up later.